addons/s3/models.py

```diff
@@ -135,10 +158,16 @@ def serialize_waterbutler_credentials(self):
         }
 
     def serialize_waterbutler_settings(self):
+        """
+        We use the folder id to hold the bucket location
```

Comment:
Minor, and I don't really know the best place for this one, but related to the comment: 

Using `folder_id` for the bucket and `folder_path` for the... folder path would be more consistent other addon behavior. What was the rationale for not doing so -- avoiding a migration?